docs/howto/add_activable_element_from_selection.md

```diff
@@ -0,0 +1,69 @@
+# How-to: Add activable element in a selection form
+
+The Hybrid Platform UI application provides a custom element called `ez-selection-activate-element`. This component gives the possibility to easily implement activable
+elements (like buttons) that will be enabled and can triggers actions only if some elements in a
```

Comment:
elements (like buttons) that will be enabled and can trigger actions only if some elements in a